libraries/tock-tbf/src/types.rs

```diff
@@ -213,6 +245,35 @@ pub struct TbfHeaderV2KernelVersion {
     minor: u16,
 }
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum TbfFooterV2CredentialsType {
+    Padding = 0,
```

Comment:
I think `Padding` should be split out into its own TLV as it's not a credential and could be useful for footers more generally.

Reply:
Do we want padding anywhere except a Footer? I figured that  since it's variable length its formatting and parsing is almost identical to a credential, and so it would be cheaper to have the same parsing code.

Reply:
Maybe, but I was counting on the unknown TLV code to work for my hypothetical padding TLV so we wouldn't need any extra code for it.

Reply:
I think moving to reserved (https://github.com/tock/tockloader/pull/86) is a good solution to this question.

However, TBFs have a requirement that all TLVs must start on a multiple of 4 bytes. So we technically need to validate that the end of app binary leaves us at a multiple of 4, and that if a reserved TLV is not a multiple of 4 there are padding bytes added if another TLV follows the reserved TLV.